### PR TITLE
Add docs for configuring babeljs

### PR DESCRIPTION
**Pull request by cibernox**
Has an example of how to disable certain ES6 transformations and links to babel 5.X docs for a comprehensive list of options and transformations.

### CODE REVIEW

**Comment by cibernox** [comment]
Part of Quest in https://github.com/ember-cli/ember-cli.github.io/issues/111

File: _posts/2013-04-09-asset-compilation.md [hunk 1]
```diff
@@ -17,6 +17,46 @@ templates with `assets/images/logo.png` or in stylesheets with
 
 This functionality of Ember-CLI comes from [broccoli-asset-rev](https://github.com/rickharrison/broccoli-asset-rev). Be sure to check out all the options and usage notes.
 
+### JS Transpiling
+
+Ember-cli automatically transpiles future javascript (ES6/ES2015, ES2016 and beyond) into standard ES5
```
Comment (stefanpenner) [inline _posts/2013-04-09-asset-compilation.md:22]: - [ ] We should briefly outline that this is using babel 5 (i saw you provided links to the appropriate docs, awesome)
- [ ]  We should point to the babel 6 meta issue, for progress getting to babel 6. For those who will likely be immediately curious

**Review by stefanpenner — CHANGES_REQUESTED** [review]

**Comment by stefanpenner** [comment]
looks good, some small notes that I think will clear up a few  remaining bits, and this may be good to 🚢

File: _posts/2013-04-09-asset-compilation.md (hunk 1 above)
Comment (cibernox) [reply _posts/2013-04-09-asset-compilation.md:22]: @stefanpenner [This](https://github.com/ember-cli/ember-cli/issues/5015) issue?

**Review by stefanpenner — APPROVED** [review]

**Comment by stefanpenner** [comment]
@cibernox awesome, thanks sir!

**Pull request merged**